Makefile

```
@@ -1,5 +1,16 @@
+check: pre-commit test
```

checkmake (0.3.2)

[warning] Target "check" should be declared PHONY.

(phonydeclared)